.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I guess this file was added by accident? `.vscode` should be added to .gitignore

Reply:
Seems on purpose to me: https://github.com/isaqb-org/advanced-template/commit/bed5bef4622e57ad455b1acc7786893ba05f8159

Reply:
Apparently, yes....